realtime_ai_character/llm/__init__.py

```diff
@@ -6,20 +6,32 @@
 from realtime_ai_character.llm.base import LLM
 
 
-def get_llm(model='gpt-3.5-turbo-16k') -> LLM:
+def get_llm(model="gpt-3.5-turbo-16k") -> LLM:
+    # temporaryly get url from .env LOCAL_LLM_URL
+    # need figure out how to set up model=url in frontend
+    # if select "Llama-2-70b" button from frontend,
+    # model here will be "meta-llama/Llama-2-70b-chat-hf"
+    model = os.getenv('LOCAL_LLM_URL')
```

Comment:
@pycui Thank you! Have made some updates based on your suggestions.
If I select "Llama-2-70b" button from frontend, `model` here will be "meta-llama/Llama-2-70b-chat-hf".
Thus I load `model` temporarily from .env here.

Reply:
Thanks, I made some change to still use the model param. For testing, you can modify the frontend to pass localhost as model name.